scripts/create-index-pages.js

```diff
@@ -0,0 +1,91 @@
+const fs = require('fs').promises;
+const fsSync = require('fs');
+const path = require('path');
+const fm = require('front-matter');
+
+const logger = require('./utils/logger');
+const { BASE_DIR } = require('./constants');
+
+const getTitle = (file) =>
+  file
+    .split('/')
+    .slice(-1)[0]
+    .replace('.mdx', '')
+    .split('-')
+    .map((word) => word.charAt(0).toUpperCase() + word.slice(1))
+    .join(' ');
+
+const getFrontmatter = (title) => `---
+title: ${title}
+template: basicDoc
+---
+
+`;
+
+const getPageContent = (dir, files) => {
+  return files
+    .map((file) => {
+      const path = `${dir}/${file.name}`
+        .replace(BASE_DIR, '')
+        .replace('.mdx', '');
+      const fileContent = fsSync.readFileSync(`${dir}/${file.name}`, 'utf8');
```

Comment:
@zstix @roadlittledawn attempted to have it read from the actual title! appears to be working

Reply:
yep, looks good to me! good stuff. i like that we maintain a fallback to the file name, too.